project/Dependencies.scala

```diff
@@ -302,6 +305,10 @@ object Dependencies {
     "org.mock-server" % "mockserver-netty" % mockserverNettyV % Test
   ) ++ betterFilesDependencies
 
+  val wdlModelDraft3Dependencies = List(
+    "com.github.mpilquist" %% "simulacrum" % simulacrumV
```

Comment:
How come this isn't coming in via `womDependencies`?

Reply:
Oh it probably is.  I added it here first.  I'll remove it